src/GLView.cc

```diff
@@ -312,6 +305,11 @@ void GLView::initializeGL()
   glEnable(GL_BLEND);
   glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
 
+#if #DEBUG
+  glEnable              ( GL_DEBUG_OUTPUT );
+  glDebugMessageCallback( MessageCallback, 0 );
```

Comment:
This is an OpenGL 4 function, and will crash under OpenGL <4

Reply:
Oops, I just copied an example from an OpenGL wiki without thinking much.  It was kind of helpful for me during debugging, but I didn't realize it was tied to such a high-ish version. 
And yeah I hastily put those busted if directives in there just before commiting.

Should I 1) just remove it, 2) comment it out with a note in case any dev wants to temporarily use it, or 3) try to wrap it in a proper version availability check?

Reply:
Commenting is fine for now. I'm not actually sure why Qt for Mac doesn't ever select a newer OpenGL version..